WalletWasabi.Tests/UnitTests/WabiSabi/Integration/WabiSabiHttpApiIntegrationTests.cs

```diff
@@ -126,6 +130,78 @@ public async Task SoloCoinJoinTestAsync(long[] amounts, long[]? outputs = null)
 			await roundStateUpdater.StopAsync(CancellationToken.None);
 		}
 
+		[Fact]
+		public async Task MultiClientsCoinJoinTestAsync()
+		{
+			const int NumberOfParticipants = 20;
+			const int NumberOfCoinsPerParticipant = 2;
+			int expectedInputNumber = NumberOfParticipants * NumberOfCoinsPerParticipant;
+
+			var node = await TestNodeBuilder.CreateForHeavyConcurrencyAsync();
+			var rpc = node.RpcClient;
+
+			var httpClient = _apiApplicationFactory.WithWebHostBuilder(builder =>
+			{
+				builder.ConfigureServices(services =>
+				{
+					// Instruct the coodinator DI container to use these two scoped
+					// services to build everything (wabisabi controller, arena, etc)
+					services.AddScoped<IRPCClient>(s => rpc);
+					services.AddScoped<WabiSabiConfig>(s => new WabiSabiConfig
+					{
+						MaxRegistrableAmount = Money.Coins(500m),
+						MaxInputCountByRound = expectedInputNumber,
+						ConnectionConfirmationTimeout = TimeSpan.FromSeconds(20 * expectedInputNumber),
+						OutputRegistrationTimeout = TimeSpan.FromSeconds(20 * expectedInputNumber),
+					});
+				});
+			}).CreateClient();
+
+			// Create the API client
+			var apiClient = _apiApplicationFactory.CreateWabiSabiHttpApiClient(httpClient);
+
+			// At the end of the test a coinjoin transaction has to be created and broadcasted.
+			var transactionCompleted = new TaskCompletionSource<Transaction>();
```

Comment:
IMO this is not used anymore, just a leftover.

Reply:
thanks